src/async_fuse/memfs/s3_node.rs

```diff
@@ -21,15 +21,11 @@ use nix::sys::stat::SFlag;
 use smol::lock::RwLock;
 use std::collections::BTreeMap;
 use std::collections::VecDeque;
+use std::os::unix::io::RawFd;
```

Comment:
Why do we reformat the use in this matter?

Reply:
I don't have strong opinion on this. I think it is more clearer. If we use old style. We need compress  all std::{} together. I have never seen this style.

Reply:
I prefer the previous version as it requires less typing and the organization is more clear, but I still didn't get my vscode work it out automatically.